spyder/plugins/preferences/widgets/configdialog.py

```diff
@@ -175,15 +248,58 @@ def resizeEvent(self, event):
         main application
         """
         QDialog.resizeEvent(self, event)
-        self.size_change.emit(self.size())
+        self.sig_size_changed.emit(self.size())
 
     @property
     def _stylesheet(self):
-        css = qstylizer.style.StyleSheet()
+        # Use special tabbar stylesheet for as the base one and then extend it.
```

Comment:
```suggestion
        # Use special tabbar stylesheet as the base one and then extend it.
```
And maybe creating a `PreferencesTabBarStyleSheet` with the changes could be worthy instead of the extension in place done here?